packages/docs/src/components/Search.jsx

```diff
@@ -53,19 +51,51 @@ export function Search(props) {
     }
   }, [])
 
-  const oramaWhere = currentCategory ? {
-    category: {
-      eq: currentCategory
-    }
-  } : {}
+  const oramaWhere = currentCategory
+    ? {
+        category: {
+          eq: currentCategory
+        }
+      }
+    : {}
```

Comment:
not sure, but maybe we don't need really need this if it's used for facets

- [ ] check if we need to add add this `where`

Reply:
checked that we need it to filter between open source and cloud doc